app/components/Layout/Footer.jsx

```diff
@@ -108,10 +108,20 @@ class Footer extends React.Component {
 
         var hintData = document.querySelectorAll("[data-intro]");
         var theme = SettingsStore.getState().settings.get("themes");
-        let scrollableDiv = document.getElementsByClassName(
-            "grid-block vertical no-padding ps-container"
+        const scrollableDiv =
+            document.getElementsByClassName(
+                "grid-block vertical no-padding ps-container"
+            )[0] || false;
+        const chartDiv = document.getElementsByClassName(
```

Comment:
This way of fixing the issue would require a custom solution for *every* panel that uses intro.js.